templates/zerver/api/changelog.md

```diff
@@ -20,6 +20,15 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 6.0
 
+Feature levels 157-159 are reserved for future use in 6.x maintenance
+releases.
+
+## Changes in Zulip 5.0
```

Comment:
No, this is the 6.0 section, and the preceding one is the 7.0 section.

Reply:
Fixed 5.0=>6.0, thanks. Removing 7.0 as noted elsewhere.